third_party/blink/renderer/farbling/DEPS

```diff
@@ -0,0 +1,8 @@
+include_rules = [
+  "+third_party/abseil-cpp/absl/random",
+  "+third_party/blink/public/platform",
+  "+third_party/blink/public/common",
+  "+third_party/blink/renderer/core",
```

Comment:
it you want to access core from here then this needs to be in core so should be third_party/blink/renderer/core/farbling

Reply:
Fixed